parachain/runtime/runtime-tests/src/relaychain/mod.rs

```diff
@@ -1,4 +1,4 @@
-mod kusama_cross_chain_transfer;
-pub mod kusama_test_net;
-mod polkadot_cross_chain_transfer;
-pub mod polkadot_test_net;
+// mod kusama_cross_chain_transfer;
+// pub mod kusama_test_net;
+// mod polkadot_cross_chain_transfer;
+// pub mod polkadot_test_net;
```

Comment:
Revert, why did you delete these files?

Reply:
While reverting, it found out that `decl_test_parachains` macro requires `palances_pallet` which is not present in interlay/kintsugi runtime.
[Macro definition](https://github1s.com/paritytech/cumulus/blob/polkadot-v1.0.0/xcm/xcm-emulator/src/lib.rs#L497)

Reply:
Issue #1190 will resolve it.